Ginger/GingerCore/Actions/Act.cs

```diff
@@ -141,10 +141,17 @@ public new static partial class Fields
             public static string SupportSimulation = "SupportSimulation";
         }
 
+        public ActionLogConfig ActionLogConfig { get; set; }
```

Comment:
Add unit test for save/load - for now it will not work as nothing is marked serialzied

Reply:
added - but ActionConfigLog, the values are not retained in the xml file. @yaronweiss, please check.